core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1452,6 +1452,13 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey CLIENT_META_DATA_CACHE_SIZE =
+      new Builder("Client.MetadataCacheSize")
+          .setDescription("Size of cached metadata. Only valid if the filesystem is"
```

Comment:
How about change to: The total number of files and directories whose metadata is cached on the client-side.